configs/config.go

```diff
@@ -19,13 +16,17 @@ type Configuration struct {
 }
 
 func (c *Configuration) GetConfiguration() *Configuration {
-	yamlFile, err := ioutil.ReadFile(ConfigurationPath)
+	viper.SetConfigName("beatshipper-conf")
+	viper.AddConfigPath("/etc/beatshipper/")
+	viper.AddConfigPath("$HOME/.config")
```

Comment:
This will not work, as the $HOME var is only expanded in bash.
To get user's home dir in golang, try: 
`dirname, err := os.UserHomeDir()`

Reply:
I wonder why... at the end `os.UserHomeDir()` get the `HOME` variable and access to it through `Getenv`.

Anyway, you are right, if we wanna use this in other different runtimes and expand the usage we can even go direct to `os.UserConfigDir()` that append the `.config` folder automatically in UNIX based distributions.